utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1016,4 +1021,95 @@ inline void compare_output(T* output, string funcName, RpptDescPtr srcDescPtr, R
         qaResults << status << std::endl;
         qaResults.close();
     }
+}
+
+// Used to randomly swap values present in array of size n
+inline void randomize(unsigned int arr[], unsigned int n)
+{
+    // Use a different seed value each time
+    srand (time(NULL));
+    for (unsigned int i = n - 1; i > 0; i--)
+    {
+        // Pick a random index from 0 to i
+        unsigned int j = rand() % (i + 1);
+        std::swap(arr[i], arr[j]);
+    }
+}
+
+// Generates a random value between given min and max values
+int inline randrange(int min, int max)
+{
+    if(max < 0)
+        return -1;
+    return rand() % (max - min + 1) + min;
+}
+
+void inline init_ricap_qa(int width, int height, int batchSize, Rpp32u *permutationTensor, RpptROIPtr roiPtrInputCropRegion)
```

Comment:
Again @snehaa8 Could you please add one comment line above this function stating what it does - like "RICAP Input Crop Region initializer for QA testing and golden output match"